mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -472,16 +477,22 @@ public void onStopTrackingTouch(SeekBar seekBar) {
                 break;
             case TYPE_IMAGE:
                 MySmartImageView imageImage = (MySmartImageView) widgetView.findViewById(R.id.imageimage);
-                imageImage.setImageUrl(ensureAbsoluteURL(openHABBaseUrl, openHABWidget.getUrl()), false,
+                OpenHABItem item = openHABWidget.getItem();
+                if (item != null && item.getType().equals("Image") && item.getState() != null
+                        && item.getState().startsWith("data:")) {
+                   imageImage.setImage(new MyImageFromItem(item.getState()));
+                } else {
+                    imageImage.setImageUrl(ensureAbsoluteURL(openHABBaseUrl, openHABWidget.getUrl()), false,
                         openHABUsername, openHABPassword);
+                    if (openHABWidget.getRefresh() > 0) {
```

Comment:
No refresh rate for the other type? Imagine following setup:
A selection that controls which webcam I want to see. The webcams make every few seconds a picture.

Reply:
Each update of an item on the server side triggers a refresh of the page in openHAB app. That's why a specific refresh of an image element linked to an item of type image is not useful.
The refresh has a sense only when using an URL for the picture with a potential change of the picture behind the URL. It is the case for example if your item is of type String and contains an URL.

Sorry but I don't understand why your mean with selection of webcams. Can you provide items and sitemap example to explain your case ?